pkg/webui/components/notification/notification.styl

```diff
@@ -63,3 +63,19 @@ left-border($color)
   .icon
     display: flex
     margin-right: $cs.xl - $border-width
+
+  .action
+    display: flex
+    width: 30%
+    min-width: 100px
+    align-items: center
+    flex-grow: 2
+    justify-content: flex-end
+
+    &-button
+      width: 10%
+
+    &-message
+      width: 50%
+      color: $tc-subtle-gray
+      font-size: $fs.s
```

Comment:
I think you can do all of this a lot simpler if you wrapped the content of the notification around a div and then display the text and the button next to each other as inline. Regard the button as part of the content.